src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const personPartnerText = isMarried
+    ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
**Never** use nester ternary :)

Reply:
So it's better just to use if else?

Reply:
u can save result from one of ternary to const